test/infrastructure/docker/config/crd/kustomization.yaml

```
@@ -1,6 +1,7 @@
 labels:
 - pairs:
-    cluster.x-k8s.io/v1beta1: v1beta1_v1beta2
+    cluster.x-k8s.io/v1beta1: v1beta1
+    cluster.x-k8s.io/v1beta2: v1beta2
```

```suggestion
    # Note: This is needed so the topology reconciler can figure out
    # the contract of v1beta1 when v1beta1 objects are used in ClusterClasses.
    cluster.x-k8s.io/v1beta1: v1beta1
    cluster.x-k8s.io/v1beta2: v1beta2
```
Let's add this comment like we have for CABPK / KCP so we know why we need both labels